.rspec

```diff
@@ -1 +1,2 @@
 --require spec_helper
+--format documentation
```

Comment:
While I like the documentation format I think it's best to set this in your home directory `.rspec` file. See https://relishapp.com/rspec/rspec-core/docs/configuration/read-command-line-configuration-options-from-files for more info.

Reply:
Done